courses/signals.py

```diff
@@ -0,0 +1,27 @@
+"""
+Signals for micromasters course certificates
+"""
+from django.db import transaction
+from django.db.models.signals import post_save
+from django.dispatch import receiver
+
+from courses.models import CourseRunCertificate
+from courses.utils import generate_program_certificate
+
+
+@receiver(
+    post_save,
+    sender=CourseRunCertificate,
+    dispatch_uid="courseruncertificate_post_save",
+)
+def handle_create_course_run_certificate(
```

Comment:
I was testing and my program certificate nor the enrollment was getting generated automatically. I dug in a bit and I noticed that this signal isn't being called at all. Further debugging I noticed that these signals are not initialized at all which made this signal not work. I'm not sure how you were able to test it.

You have to initialize the courses.signals in `ready` callback courses app. Something like :
```
    def ready(self):
        import courses.signals
```